stdlib/experimental/http/requests/requests.flux

```diff
@@ -211,3 +284,28 @@ get = (
         body: body,
         config: config,
     )
+
+// peek converts a response into a table for easy inspection.
+// The response body will be returned as a string column on the table.
+// The status code is returned as an integer column on the table.
+//
+// Flux queries must produce a table this method makes that conversion easy.
+// If however you want control over how the table is constructed use `array.from`
+// in combination with a function like `json.parse`. The basic process is parse the
+// response body into a set of values and then use `array.from` to construct a 
+// table from those values. See the examples for `requests.post` and `request.get` for
+// a more complete explanation.
```

Comment:
```suggestion
// To customize how the response data is structured in a table, use `array.from()`
// with a function like `json.parse()`. Parse the response body into a set of values
// and then use `array.from()` to construct a table from those values.
```

Reply:
I'm wondering if we should even include this content or at least move it elsewhere. It's essentially telling a user how to not use `requests.peek`.

Reply:
Yeah my thinking was the other examples have taught users how to use `peek` but they will need to learn how to move beyond using just `peek` as its not that useful beyond debugging. Happy to move this around as needed.

Reply:
It's probably fine to leave it here.